angular-cli-build.js

```diff
@@ -15,7 +15,8 @@ module.exports = function(defaults) {
       'es6-shim/es6-shim.js',
       'reflect-metadata/**/*.+(ts|js|js.map)',
       'rxjs/**/*.+(js|js.map)',
-      '@angular/**/*.+(js|js.map)'
+      '@angular/**/*.+(js|js.map)',
+      '@angular2-material/**/*'
```

Comment:
What is the benefit of adding angular material? Where are the docs and getting started with the material? Please share these.